setup.sh

```diff
@@ -197,6 +197,22 @@ function setupModule() {
     return
   fi
 
+  if [[ ${2} == "is" ]]; then
+    checkoutBranch ${1} ${4}
+    checkoutTag ${1} ${5}
+    mv "${PUPPET_HOME}/modules/${1}" "${PUPPET_HOME}/modules/${1}-tmp"
+    mv "${PUPPET_HOME}/modules/${1}-tmp/wso2is" "${PUPPET_HOME}/modules/"
+    mv "${PUPPET_HOME}/modules/${1}-tmp/wso2is_analytics" "${PUPPET_HOME}/modules/"
+    rm -rf "${PUPPET_HOME}/modules/${1}-tmp"
```

Comment:
Why the content is moved to a tmp and back to original path ?

Reply:
This is because here the name of the checkout path of puppet-is content is also 'wso2is'. Therefore, cannot copy 'wso2is' puppet module inside that one to the upper directory as it already has a directory as 'wso2is'.
This is why content is moved to a tmp path and the modules inside the tmp path is copied to modules/ path.